hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AUtils.java

```diff
@@ -659,11 +678,12 @@ public static AWSCredentialProviderList buildAWSProviderList(
             + " in option " + key + ": " + aClass);
       }
 
-      // TODO: Not sure what to do here yet.
-      //  There were some V1 providers for which we suppressed warnings, how do we map those?
-      if (aClass.getName().contains(AWS_AUTH_CLASS_PREFIX)) {
-        // V2Migration.v1ProviderReferenced(aClass.getName());
-
+      if (Arrays.asList("EnvironmentVariableCredentialsProvider",
```

Comment:
Can't you just check that simpleName is in the map?

Reply:
yup! have updated